plugins/filter_aws/aws.c

```diff
@@ -558,22 +584,14 @@ static int cb_aws_filter(const void *data, size_t bytes,
                                   ctx->availability_zone_len);
         }
 
-        if (ctx->instance_id_include && !ctx->enable_entity) {
```

Comment:
So if `instance_id_include` and `enable_entity` are both on, we will `msgpack` the normal `instance_id_include` parameters (e.g. `FLB_FILTER_AWS_INSTANCE_ID_KEY_LEN`) and then also `msgpack` the `aws_entity` versions (e.g. `FLB_FILTER_AWS_ENTITY_INSTANCE_ID_KEY_LEN`? 
Whereas before this change we would only `msgpack` the one that was appropriate?

Why are we making this change?

Reply:
This is done so that entity fields are independent of the `instance_id_include` fields. If customer wants to have instance id in the log message, while still sending entity, this change help with that.